modules/ROOT/pages/runtime-urls-whitelist.adoc

```diff
@@ -16,16 +16,18 @@ The following table shows you the ports or IPs/hostnames to add to your whitelis
 |*US*|*analytics-ingest.anypoint.mulesoft.com* |  443
 |*US*|*arm-auth-proxy.prod.cloudhub.io* |  443
 |*US*|*data-authenticator.anypoint.mulesoft.com* |  443
+|*US*|*exchange-files.anypoint.mulesoft.com* |  443
 |*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443
 |*EU*|*eu1.anypoint.mulesoft.com* | 443
 |*EU*|*mule-manager.eu1.anypoint.mulesoft.com* | 443
 |*EU*|*runtime-manager.eu1.anypoint.mulesoft.com* | 443
 |*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443
 |*EU*|*arm-auth-proxy.prod-eu.cloudhub.io* |  443
 |*EU*|*data-authenticator.eu1.anypoint.mulesoft.com* |  443
+|*US*|*exchange-files.eu1.anypoint.mulesoft.com* |  443
```

Comment:
I think exchange-files.eu1.anypoint.mulesoft.com is for EU region, not US.

Reply:
Fixed